### ISSUE TITLE
[Feature] Hide/delete articles with non latin script

### ISSUE THREAD

**Issue by lamyergeier** [FreshRSS/FreshRSS#6144]
**Is your feature request related to a problem? Please describe.**
I would like to see feeds in English or German language that is I don't want to see feeds in Chinese, Japanese, Korean, Arabic, Thai, Hindi, Tamil, Kannada, Telugu languages. But there are some channels which occasionally post in these languages. 

**Describe the solution you’d like**
How to filter out and mark these as read (or if possible delete it)?

**Describe alternatives you’ve considered**

**Additional context**

**Comment by math-GH** [comment]
Could you support please with some example feeds?

**Comment by lamyergeier** [comment]
Please check this YouTube channel video

https://www.youtube.com/watch?v=k4R7ROY0amk


This channel usually post videos in English.
----------------------------------------

Mar 2, 2024 14:10:26 maTh ***@***.***>:

**Comment by math-GH** [comment]
That is the matching RSS feed: https://www.youtube.com/feeds/videos.xml?channel_id=UCXQZTtGgsy6QHH2fg-OJ_eA

**Comment by lamyergeier** [comment]
Yes, so I would like to hide articles that have non latin characters (from above mentioned languages)

**Comment by math-GH** [comment]
try this for `Filter applied: Mark an article as read…`:

`!intitle:a !intitle:i !intitle:e !intitle:o !intitle:u`

Does it solve your problem?

**Comment by lamyergeier** [comment]
I understand the logic that there should be atleast one of the vovel in the English Title. This should work. Hopefully they would implement more rigorous filter option.
